ports/mimxrt/hal/pwm_backport.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * Copyright (c) 2015, Freescale Semiconductor, Inc.
+ * Copyright 2016-2017 NXP *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * The MIT License (MIT)
```

Comment:
same thing re licensing as for the .c file

Reply:
OK